code/style.css

```diff
@@ -1,7 +1,214 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #fff;
+}
+
+.header-section {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+video {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  z-index: -1;
+}
+
+.hero-image-section {
+  margin: 0;
+}
+
+/* Paragraph and text  */
+
+h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  width: 280px;
+  text-align: center;
+  margin-bottom: 150px;
+}
+
+h3 {
+  font-size: 2rem;
+  font-weight: 600;
+  margin-top: 0;
+}
+
+.questions {
+  font-size: 1.25rem;
+  font-weight: 400;
+  margin: 0;
+}
+
+p {
+  font-size: 0.98rem;
+  font-weight: 300;
+  line-height: 1.8rem;
+}
+
+hr {
+  margin: 35px 0;
+}
+
+/* Form */
+
+.form-section {
+  background-image: linear-gradient(135deg, #52e5e7, #130cb7);
```

Comment:
I love your gradient background, you made it look sooo professional! Want to apply it as well. Great job!